analyzer/src/funTest/assets/projects/synthetic/all-managers/environment.yml

```diff
@@ -0,0 +1,6 @@
+name: ort
+channels:
+  - defaults
+dependencies:
+  - python=3.7
+  - django==2.2.1
```

Comment:
This file should be empty because it is only used to test if the definition files are correctly detected.

Reply:
emptied file